web/client/components/data/query/GeometryDetails.jsx

```diff
@@ -77,7 +79,7 @@ class GeometryDetails extends React.Component {
             projection: this.props.geometry.projection
         };
 
-        this.props.onChangeDrawingStatus("replace", undefined, "queryform", [geometry]);
+        this.props.onChangeDrawingStatus({geometry: [geometry]});
```

Comment:
is CHANGE_DRAWING_STATUS without a "command" (e.g. replace) supported?

Reply:
maybe was better to change the name, but isn't directly calling the onChangeDrawingStatus action. That action is debounced and then called by enhancer

Reply:
I mean, if you put a "transparent" debounce in the middle, it should be transparent, if I remove it the same code will work, without debouncing. We want to enhance, not to change.

Reply:
Actions have a standard interface, you are changing it here